src/util/sync-location-hashes.test.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-env jest */
+
+import syncLocationHashes from 'src/util/sync-location-hashes'
+
+function createWindowStub({hash}) {
+    return [{
+        location: {hash},
+        addEventListener: jest.fn(),
+        removeEventListener: jest.fn(),
+    }]
+}
```

Comment:
I meant to accept two hashes as arguments and return this:
```
    return [{
        location: {hash1},
        addEventListener: jest.fn(),
        removeEventListener: jest.fn(),
    }, {
        location: {hash2},
        addEventListener: jest.fn(),
        removeEventListener: jest.fn(),
    }]
```

Besides possibly needing this in tests we should still add later, passing two (or more) windows just seems a more plausible example of how the code is used in practice.